packages/legacy/core/App/screens/UseBiometry.tsx

```diff
@@ -153,9 +238,9 @@ const UseBiometry: React.FC = () => {
             <ToggleButton
               testID={testIdWithKey("ToggleBiometrics")}
               isEnabled={biometryEnabled}
-              isAvailable={biometryAvailable}
+              isAvailable={true}
```

Comment:
is it ok that these are hard coded values? the diff shows `biometryAvailable`

Reply:
Yes, it was changed so that the biometric switch is now always enabled for interaction, it will shows a popup with messages when biometryAvailable == false; instead of being disabled from interacting entirely before this change.